src/utils/ICS20Lib.sol

```diff
@@ -25,17 +25,19 @@ library ICS20Lib {
     }
 
     /// @notice Convenience type used after unmarshalling the packet data and converting addresses
-    /// @param denom The denom of the token
+    /// @param ibcDenom The ibc denom of the token
+    /// @param fullDenomPath The full denom path of the token
     /// @param originatorChainIsSource True if origniating chain is source of token
-    /// @param erc20Contract The address of the ERC20 contract
+    /// @param erc20Address The address of the ERC20 contract
     /// @param amount The amount of tokens
     /// @param sender The sender of the tokens
     /// @param receiver The receiver of the tokens
     /// @param memo Optional memo
     struct UnwrappedPacketData {
```

Comment:
This struct keeps getting more complicated and it already has a weird name. I think we might need to think of another way to pass information (instead of adding new fields to this struct).

Reply:
Yeah, I mentioned this struct in the refactor issue: https://github.com/srdtrk/solidity-ibc-eureka/issues/36